steps/compliance/mrn-date/content.cy.json

```diff
@@ -4,11 +4,11 @@
   "dateLegend": "Nodwch y dyddiad sydd yng nghornel dde uchaf eich llythyr MRN",
   "dateHint": "Er enghraifft, 31 3 2017",
   "imageAlt": "Copi o’ch MRN",
-  "titleHeadIba": "IRN dated",
-  "titleIba": "When is your Internal Review Notice (IRN) dated?",
-  "dateLegendIba": "Enter the date from the top right of your IRN letter",
-  "dateHintIba": "For example, 26 1 2024",
-  "imageAltIba": "A copy of your IRN",
+  "titleHeadIba": "Review Decision Notice dated",
+  "titleIba": "When is your Review Decision Notice dated?",
+  "dateLegendIba": "Enter the date from the top right of your Review Decision Notice",
+  "dateHintIba": "Er enghraifft, 26 1 2024",
```

Comment:
Line is in English in the original version. Is this correct?

Reply:
We are waiting on the Welsh translation.